drivers/ec.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Lightweight ACPI Implementation
+ * Copyright (C) 2018-2019 the lai authors
+ */
+
+/* LAI Embedded Controller implementation
+ * ACPI 6.3 Specification chapter 12
+ * ACPI defines an Embedded Controller for interfacing directly with firmware
+ */
+
+#include <lai/drivers/ec.h>
+
+#define ACPI_EC_PNP_ID "PNP0C09"
+
+static uint16_t ec_cmd_port = 0;
+static uint16_t ec_data_port = 0;
+
+void lai_init_ec(void){
```

Comment:
We should not enumerate all ECs in this function. Instead, it should take a `lai_nsnode_t *` handle of the EC node. We can move enumeration to a helpers (it's useful even outside of the driver).